### DISCUSSION TITLE
alist挂载阿里云open之后访问word、pdf等文件无法使用aliyun的预览

### DISCUSSION THREAD

**Discussion by JiaLeDL** [AlistGo/alist#3685] [Q&A]
![image](https://user-images.githubusercontent.com/124766265/222433300-595de77c-c621-4220-adf0-71dbc4ca08db.png)
用最新版本的alist 然后挂载的是阿里云盘open的  挂载之后  网页点击word、pdf等文件不能调用阿里云的预览

**Comment by xinhao8000** [comment]
同样的问题

**Comment by III1III** [comment]
这个默认用的是微软的在线预览，但是奇怪的是挂载阿里云盘的文件都预览不了，尝试用了Google、onlyoffice也不行，挂载其他云盘的文件可以正常预览，结果看了一下发现预览用的文件地址是阿里的api，不知道是不是做了防盗链。
<img width="1511" alt="image" src="https://user-images.githubusercontent.com/43006765/224372802-ce8291b4-1ab3-4f5a-beac-10fd854e2335.png">

**Reply by zhushen12580** [reply]
> 这个默认用的是微软的在线预览，但是奇怪的是挂载阿里云盘的文件都预览不了，尝试用了Google、onlyoffice也不行，挂载其他云盘的文件可以正常预览，结果看了一下发现预览用的文件地址是阿里的api，不知道是不是做了防盗链。 <img alt="image" width="1511" src="https://user-images.githubusercontent.com/43006765/224372802-ce8291b4-1ab3-4f5a-beac-10fd854e2335.png">

iframe变量$e_url改成$durl

**Reply by TheStoneFish** [reply]
> > 这个默认用的是微软的在线预览，但是奇怪的是挂载阿里云盘的文件都预览不了，尝试用了Google、onlyoffice也不行，挂载其他云盘的文件可以正常预览，结果看了一下发现预览用的文件地址是阿里的api，不知道是不是做了防盗链。 <img alt="image" width="1511" src="https://user-images.githubusercontent.com/43006765/224372802-ce8291b4-1ab3-4f5a-beac-10fd854e2335.png">
> 
> iframe变量$e_url改成$durl

我改成durl还是不能预览

**Comment by anwen-anyi** [comment]
阿里云Open 短期内开放不了这个Api了😭😭
问过了已经这个是属于公司其他团队了产品一时半会儿开放不了

**Reply by III1III** [reply]
真是个悲伤的故事，现在我的解决方案是alist+rclone+cloudreve做了一下中转，文件太多不想迁移，就是比官方api慢了一丢丢，偶尔看下文件改下文件够用了。

**Comment by zhushen12580** [comment]
不是的呀，我的onlyoffice和围绕都可以呀







------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年4月3日(星期一) 中午11:50
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [alist-org/alist] alist挂载阿里云open之后访问word、pdf等文件无法使用aliyun的预览 (Discussion #3685)





   
这个默认用的是微软的在线预览，但是奇怪的是挂载阿里云盘的文件都预览不了，尝试用了Google、onlyoffice也不行，挂载其他云盘的文件可以正常预览，结果看了一下发现预览用的文件地址是阿里的api，不知道是不是做了防盗链。 
  
iframe变量$e_url改成$durl
  
我改成durl还是不能预览

**Comment by TheStoneFish** [comment]
好吧我是阿里云open&nbsp;就这个open的路径有问题那个路径是一长串


---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年4月3日(周一) 中午11:56
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [alist-org/alist] alist挂载阿里云open之后访问word、pdf等文件无法使用aliyun的预览 (Discussion #3685)




 不是的呀，我的onlyoffice和围绕都可以呀 
 
 
 
 
 
 
 
 ------------------&amp;nbsp;原始邮件&amp;nbsp;------------------ 
 发件人: ***@***.***&amp;gt;;  
 发送时间: 2023年4月3日(星期一) 中午11:50 
 收件人: ***@***.***&amp;gt;;  
 抄送: ***@***.***&amp;gt;; ***@***.***&amp;gt;;  
 主题: Re: [alist-org/alist] alist挂载阿里云open之后访问word、pdf等文件无法使用aliyun的预览 (Discussion #3685) 
 
 
 
 
 
     
 这个默认用的是微软的在线预览，但是奇怪的是挂载阿里云盘的文件都预览不了，尝试用了Google、onlyoffice也不行，挂载其他云盘的文件可以正常预览，结果看了一下发现预览用的文件地址是阿里的api，不知道是不是做了防盗链。  
    
 iframe变量$e_url改成$durl 
    
 我改成durl还是不能预览 
   
 — 
 Reply to this email directly, view it on GitHub, or unsubscribe. 
 You are receiving this because you commented.Message ID: ***@***.***&amp;gt; 
&mdash;
